distributions/jamulus-server.desktop.in

```diff
@@ -0,0 +1,18 @@
+[Desktop Entry]
+Name=Jamulus (server)
```

Comment:
We should add `Name[fr]=Jamulus (serveur)` here.

Reply:
done 69cf722